iothub/device/src/Transport/Mqtt/MqttTransportSettings.cs

```diff
@@ -94,10 +96,15 @@ public MqttTransportSettings(TransportType transportType)
 
         public TransportType GetTransportType()
         {
-            return this.transportType;
+            return _transportType;
         }
 
-        public TimeSpan DefaultReceiveTimeout { get; set; }
```

Comment:
We're still changing the public API; we cannot set DefaultReceiveTimeout  anymore